plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js

```diff
@@ -118,7 +118,13 @@ jQuery( function( $ ) {
 			// Init TipTip
 			$( '#tiptip_holder' ).removeAttr( 'style' );
 			$( '#tiptip_arrow' ).removeAttr( 'style' );
-			$( '.woocommerce_variations .tips, .woocommerce_variations .help_tip, .woocommerce_variations .woocommerce-help-tip', wrapper )
+			$(
+				'.woocommerce_variations .tips, ' +
+				'.woocommerce_variations .help_tip, ' +
+				'.woocommerce_variations .woocommerce-help-tip, ' +
+				'.toolbar-variations-defaults .woocommerce-help-tip',
```

Comment:
Added this `'.toolbar-variations-defaults .woocommerce-help-tip'` to make sure that the default form value tooltip was also being initialized.